web/src/NoteView/types.ts

```diff
@@ -35,10 +35,12 @@ export class Note extends Record<{
   id: string;
   type: NoteType;
   position: Position;
+  appearBefore: number;
```

Comment:
I suggest this name should show the unit of time. Milliseconds? Beat? For example, `AppearBeforeBeats: number`